Terran-One/cw-simulate-ui
# Added new JSON viewer.

## Issue link

[WL-XXX](https://trello.com/c/21jhrDmR/31-change-json-renderer)

## Description
Have replaced the old JSON viewer with react-json-view. It's sort of experimental.

## Test steps

1. Go to simulation page.
2. Execute some messages, now you can see the request/state with new json renderer.
